src/com/xilinx/rapidwright/edif/EDIFNetlist.java

```diff
@@ -182,6 +179,16 @@ public class EDIFNetlist extends EDIFName {
         }
     }
 
+    /**
+     * Map that stores EDIFCellInst to EDIFPropertyValue for non-default IOStandard
+     * values propagated from other parts of the netlist.
+     * Anecdotally, it's been observed that Vivado propagates the IOStandard property
+     * from the EDIFNet feeding a top-level output port to its driving EDIFCellInst
+     * I/O buffer.
+     */
+    Map<EDIFCellInst,EDIFPropertyValue> cellInstIOStandardFallback;
```

Comment:
This is currently package-private, only to allow the tests to clear it. Perhaps an explicit reset method should be added. The indirect getter is through `EDIFNetlist.getIOStandard()` since it's the fallback choice, I'm not yet seeing a reason why a developer would need to get this map directly since its "fallback" nature could be mistakenly forgotten about.